beep/protocol/maccor.py

```diff
@@ -1269,6 +1369,87 @@ def insert_driving_parametersv1(reg_params, waveform_directory=None):
     return file_path
 
 
+def insert_charging_parametersv1(reg_params, waveform_directory=None, max_c_rate=3.0, min_c_rate=0.2):
```

Comment:
Minor comment: what's the reason for `v1` designation for this method?

Reply:
This is a specific way of parameterizing the fast waveform. Since there might be others down the line, I am hedging a bit to allow for new functions that allow for different parameterizations. Usually for these functions, we want legacy functions to continue to work in the same way so that we can reproduce older results.